src/coffeeandtacos/TacoParser.java

```diff
@@ -8,8 +8,38 @@
     public Trackable Parse(String line) {
         
         logger.logInfo("Begin Parsing");
+        if(line == null)
+            {
```

Comment:
Make sure you're following a standard practice. Best practice is to start scope on the start line of the if statement/method/class etc.